tests/Agent/IntegrationTests/UnboundedIntegrationTests/CosmosDB/CosmosDBTests.cs

```diff
@@ -27,11 +27,10 @@ protected CosmosDBTestsBase(TFixture fixture, ITestOutputHelper output)  : base(
             _fixture.SetTimeout(TimeSpan.FromMinutes(2));
 
             _fixture.AddCommand($"CosmosDBExerciser StartAgent");
-            _fixture.AddCommand($"CosmosDBExerciser CreateReadAndDeleteDatabase test_db_{Guid.NewGuid():x}");
-
 
-            _fixture.AddCommand($"CosmosDBExerciser Wait");
```

Comment:
We no longer need to wait for an arbitrary amount of time  as the DynamicMethodInvoker can now handle Task based method.